readme.md

```diff
@@ -57,13 +57,15 @@ The following dependencies need to be installed on your system:
 
 * [Python](https://www.python.org/), version 3.7, 32 bit
 	* Use latest minor version if possible.
-* Microsoft Visual Studio 2019 Community, Version 16.3 or later:
-	* Download from https://visualstudio.microsoft.com/vs/
+* Microsoft Visual Studio 2019, Version 16.11:
```

Comment:
In order to avoid having to specify the version, and keep it in sync with AppVeyor, we can just reference AppVeyor.

```suggestion
* Microsoft Visual Studio 2019:
        * To replicate the production build environment, use the [version of Visual Studio that AppVeyor is using](https://www.appveyor.com/docs/windows-images-software/#visual-studio-2019). 
```